tools/BUILD

```diff
@@ -11,7 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-package(default_visibility = ["//visibility:private"])
+package(default_visibility = ["//visibility:public"])
```

Comment:
can we just increase visibility of the toolchain_type_docker target only

Reply:
Done